.github/workflows/increment-versions.yaml

```diff
@@ -0,0 +1,57 @@
+name: Increment Versions
+on:
+  create:
+    tags:
+    
+jobs:
+  something:
```

Comment:
I think we can get rid of the `something` job now that we've finished debugging.